coi/benches/benches.rs

```diff
@@ -31,30 +33,26 @@ fn create_positive_coi(n: usize, embedding_size: usize) -> Vec<PositiveCoi> {
                 .collect_vec()
                 .try_into()
                 .unwrap();
-            PositiveCoi::new(CoiId::new(), point)
+            PositiveCoi::new(CoiId::new(), point, now)
         })
         .collect()
 }
 
 fn bench_compute_coi_decay_factor(c: &mut Criterion) {
-    let horizon = black_box(Duration::new(3600 * 24 * 30, 0)); // 30 days
-    let now = black_box(SystemTime::now());
-    let last_view = black_box(
-        SystemTime::now()
-            .checked_sub(Duration::new(3600, 0))
-            .unwrap(),
-    );
+    let horizon = Duration::new(60 * 60 * 24 * 30, 0); // 30 days
+    let now = Utc::now();
+    let last_view = now - chrono::Duration::seconds(60 * 60);
 
     c.bench_function("compute_coi_decay_factor", |b| {
-        b.iter(|| compute_coi_decay_factor(horizon, now, last_view))
+        b.iter(|| black_box(compute_coi_decay_factor(horizon, now, last_view)))
```

Comment:
makes sense. i'll also switch to the `std::hint::black_box()` which finally got stablized with `1.66` (this could already be used in nightly in criterion via the `real_black_box` feature).